scripts/combine/pullsAndImpacts.py

```diff
@@ -199,7 +199,7 @@ def readFitInfoFromFile(rf,filename, group=False, sort=None, ascending=True, sta
     df['absimpact'] = np.abs(df['impact'])
     df['abspull'] = np.abs(df['pull'])
     if not group:
-        df.drop(df.loc[df['label'].str.contains('massShift100MeV')].index, inplace=True)
+        df.drop(df.loc[df['label'].str.contains('massShift.*100MeV', regex=True)].index, inplace=True)
```

Comment:
You may want to exclude the efficiency uncertainties split by step, the regular expression to exclude them is ".\*eff_(syst|stat)_.\*"

Reply:
Exclude them from the group plotting by default?

Reply:
Yeah, since we usually say they are annoying. I currently use a different plotting script where I can filter the lines to plot for the groups, you may have something similar depending on how often you deem they would be needed in the plot